ecl/hql/hqlgram.y

```diff
@@ -6728,6 +6730,30 @@ primexpr1
                             Owned<ITypeInfo> ltype = lexpr->getType();
                             Owned<IHqlExpression> locale = (ltype->getTypeCode() == type_varstring) ? lexpr.getLink() : createValue(no_implicitcast, makeVarStringType(ltype->getStringLen()), lexpr.getLink());
                             $$.setExpr(createValue(no_unicodeorder, makeIntType(4, true), $3.getExpr(), $5.getExpr(), locale.getLink(), $9.getExpr()));
+                         }
+    | LIKELY '(' booleanExpr ')'
+                        {
+                            parser->normalizeExpression($3);
```

Comment:
Minor: This isn't needed because you have used booleanExpr - which already has a check.